mapbox-lint/src/test/java/com/mapbox/maps/lint/LifecycleMethodDetectorTest.kt

```diff
@@ -61,7 +61,7 @@ class LifecycleMethodDetectorTest {
       .run()
       .expect(
         """
-         |src/com/foo/TestClass.java:6: Warning: No need to invoke onStart/onStop/onDestroy/onLowMemory with Mapbox Lifecycle Plugin. [Lifecycle]
+         |src/com/foo/TestClass.java:6: Warning: ${LifecycleMethodDetector.REPORT_MESSAGE} [Lifecycle]
```

Comment:
Instead of you can use `ISSUE` properties: id, description and getExplanation(TextFormat.RAW).

Reply:
Also it could be good to add `.expectWarningCount(1)` to the chain.